packages/application-shell/src/components/project-data-locale/project-data-locale.js

```diff
@@ -1,24 +1,17 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { deepEqual } from 'fast-equals';
 import { STORAGE_KEYS } from '../../constants';
 
-const defaultLocale = 'en';
-
 const getSelectedDataLocaleForProject = projectLocales => {
-  // In case the project is not present (e.g. 404), we just return `en`
-  // as a fallback value but it doesn't really matter at end what the
-  // value is. When the user access a proper project, this value will
-  // be correctly picked.
-  if (!projectLocales) return defaultLocale;
```

Comment:
I checked and this is not necessary anymore, as we don't render this if the project is not found.

https://github.com/commercetools/merchant-center-application-kit/blob/b11b3856b9d519ad9e76eb59dc213ee9872f988a/packages/application-shell/src/components/project-container/project-container.js#L153